server/src/modules/coingecko/coingecko.controller.ts

```diff
@@ -36,8 +37,9 @@ export class CoingeckoController {
   async getCoinIdMarketChart(
     @Param('id') id: string,
     @Query('days') days: number | 'max',
+    to: number, // TODO: seems unnecessary
```

Comment:
why adding `to` here with a unnecessary comment? I don't get it

Reply:
This looks like my mistake, I removed this change and aligned test suit